app/models/job_run.rb

```diff
@@ -97,6 +91,16 @@ def file_manifest
                                   staging_location: batch_context.staging_location)
   end
 
+  def human_state_name_with_preassembly_errors
+    return human_state_name unless with_preassembly_errors?
+
+    "#{human_state_name} (with preassembly errors)"
+  end
+
```

Comment:
Is there a more seamless way to do this?  Like can we just override the method state machine `human_state_name` method here and then use `super unless with_preassembly_errors?` in line 95?   Otherwise I can see being confused in the future by `job_run.human_state_name_with_preassembly_errors` conveying any possible state name.

Reply:
Changed.